### ISSUE TITLE
a11y: Color Contrast - Kubectl Shell

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13741]
Insufficient color contrast for informative image

## Actual result

The informative image associated with "Kubectl: local" text has an insufficient color contrast ratio on hover with their background making it difficult for low vision users to discern meaningful graphics that have insufficient contrast.

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Instances

"Foreground color: #3D98D3 
Background color: #F4F5FA
Contrast ratio: 2.90:1 
Main Content 
- Close interactive image"

"Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.16:1
Main content
- Clear (in default state) 

Foreground color: #5D995D
Background color: #FFFFFF
Contrast ratio: 3.39:1  
Main content
- Connected 

Foreground color: #E4D475
Background color: #FFFFFF
Contrast ratio: 1.50:1
Main content
- Connecting (available in the beginning when we open the page)  "

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

The contrast ratio between the key part of the image and its adjacent colors should be sufficient.

**Comment by rak-phillip** [comment]
Issue 146 fails VPAT:

> "Actual Result: The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.
>
> Instances:
> Foreground color: #FFE585
> Background color: #FFFFFF
> Contrast ratio: 1.3:1
> Main content
> - Connecting (available in the beginning when we open the page)  
>
> Recommendation: On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text."

Issue 147 - FIXED
